README.md

```diff
@@ -31,7 +31,7 @@ The basic idea is that the build process results in a `bundle.js` file containin
 
 
 #### Code of conduct
-Please be civil when discussing contributions to the Cay front end code and the Coral Project. If in doubt, please consult our [Code of Conduct](https://the-coral-project.gitbooks.io/coral-bible/content/codeofconduct.html)
+Please be civil when discussing contributions to the Cay front end code and the Coral Project. If in doubt, please consult our [Code of Conduct](https://github.com/coralproject/reef/blob/master/CODE-OF-CONDUCT.md).
```

Comment:
Is there a "canonical" version? I guess we should point this link at whatever gets updated the most frequently. I thought the gitbooks page was the top level CoC from which all the others originate. @losowsky @jde thoughts?